README.md

```diff
@@ -65,34 +65,38 @@ configured to start
 
 Use the `Dockerfile`'s `dev` stage to run just the application itself in a container. Use this only for local
 development (not production)! Mounting the `object_storage_api` directory to the container via a volume means that
-FastAPI will watch for changes made to the code and automatically reload the application on the fly.
+FastAPI will watch for changes made to the code and automatically reload the application on the fly. The
+application requires a MongoDB and a MinIO instance to run.
+Instances of these can be started using the `docker-compose.yml` file.
 
-1. Build an image using the `Dockerfile`'s `dev` stage from the root of the project directory:
+1. Start a MongoDB instance:
 
    ```bash
-   docker build --file Dockerfile --target dev --tag object-storage-api:dev .
+   docker compose up --detach mongo-db minio
    ```
 
-2. Start the container using the image built and map it to port `8002` locally:
+2. Once the MongoDB and MinIO containers are running, run the container for creating the MinIO buckets:
 
    ```bash
-   docker run \
-    --publish 8002:8000 \
-    --name object-storage-api \
-    --volume ./object_storage_api:/app/object_storage_api \
-    --volume ./keys/jwt-key.pub:/app/keys/jwt-key.pub \
-    object-storage-api:dev
+   docker compose up minio-create-buckets
+   ```
+
+3. Build an image using the `Dockerfile`'s `dev` stage from the root of the project directory:
+
+   ```bash
+   docker build --file Dockerfile --target dev --tag object-storage-api:dev .
    ```
 
-   or with values for the environment variables:
+4. Start the container using the image built and map it to port `8002` locally:
 
    ```bash
    docker run \
     --publish 8002:8000 \
     --name object-storage-api \
-    --env DATABASE__NAME=ims \
+    --env-file ./.env \
     --volume ./object_storage_api:/app/object_storage_api \
     --volume ./keys/jwt-key.pub:/app/keys/jwt-key.pub \
+    --volume ./logging.ini:/app/logging.ini \
     object-storage-api:dev
```

Comment:
I get `pymongo.errors.ServerSelectionTimeoutError: localhost:27017: [Errno 111] Connection refused` when I run it like this and send a `GET` request to the `/images` endpoint. I thought it needed `--add-host localhost:host-gateway \` but that did not solve the issue.

Reply:
Mine worked fine after adding the host, I can upload/get attachments and images.